en-US/mozorg/internet-health/privacy-security.lang

```diff
@@ -12,6 +15,20 @@ A Healthy Internet is Secure and Private
 The Internet only stays healthy if we trust it as a safe place – to explore, transact, connect, and create.
 
 
+## TAG: internet_health_digital_inclusion
+;Privacy &amp; Security
+Privacy &amp; Security
+
+
+## TAG: internet_health_digital_inclusion
+;Digital Inclusion
+Digital Inclusion
+
+
+## TAG: internet_health_open_innovation
```

Comment:
This string is behind the same tag as the rest of the menu (internet_health_digital_inclusion)
https://github.com/alexgibson/bedrock/blob/06448a41063b2d870df9f018324b8db2ccd3326a/bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html#L29-L37